frontend/pages/hosts/HostDetailsPage/HostDetailsPage.jsx

```diff
@@ -388,28 +390,32 @@ export class HostDetailsPage extends Component {
             </p>
           </div>
         ) : (
-          <div className={`${baseClass}__wrapper`}>
-            <table className={wrapperClassName}>
-              <thead>
-                <tr>
-                  <th>Name</th>
-                  <th>Type</th>
-                  <th>Installed Version</th>
-                </tr>
-              </thead>
-              <tbody>
-                {!!host.software.length &&
-                  host.software.map((software) => {
-                    return (
-                      <SoftwareListRow
-                        key={`software-row-${software.id}`}
-                        software={software}
-                      />
-                    );
-                  })}
-              </tbody>
-            </table>
-          </div>
+          <>
+            <SoftwareVulnerabilities softwareList={host.software} />
+            <div className={`${baseClass}__wrapper`}>
+              <table className={wrapperClassName}>
+                <thead>
+                  <tr>
+                    <th />
+                    <th>Name</th>
+                    <th>Type</th>
+                    <th>Installed Version</th>
+                  </tr>
+                </thead>
+                <tbody>
+                  {!!host.software.length &&
```

Comment:
Same thing as a previous PR. If `host.software` is always an array, we won't need to check the length since it's always zero or greater.